build/single.html

```diff
@@ -6,34 +6,79 @@
+
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Incidunt quaerat, tempore omnis quasi quam, dignissimos voluptatum ullam architecto accusantium facere sit non facilis consequuntur sint sed at, neque tempora quibusdam asperiores quisquam molestiae necessitatibus ipsum ea quia. Veniam eius non cumque eaque enim nesciunt, vel repellendus eum debitis magnam! Culpa odit quaerat dignissimos consequuntur omnis molestias rerum ut praesentium. Animi possimus maxime, minus laborum atque doloribus officiis magni qui non perferendis, neque tempore facilis iusto a voluptate hic aliquid deleniti ad explicabo aut quas eveniet asperiores nulla tenetur minima. Architecto delectus dolore quasi natus reprehenderit explicabo, magni totam, nobis tenetur.</p>
+
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsum corporis officiis, nam nostrum ipsa beatae reprehenderit aut dolor commodi dolorem consectetur optio repellendus, facere ratione minus delectus veniam placeat. Commodi excepturi, animi, fuga optio reiciendis explicabo soluta nihil nisi. Modi repudiandae officia consequatur in, laborum labore explicabo inventore quidem voluptatum eligendi architecto nihil omnis maiores animi vero, veniam excepturi, necessitatibus, totam obcaecati odio dolorum dolor error aliquam et facilis. Quibusdam ut, doloremque in cupiditate voluptate!</p>
+      </div>
+
+      <div class="side-bar">
+        <div class="mod">
+            <h2>More Articles</h2>
+            <ul>
+                <li><a href="#">Lorem ipsum dolor sit amet.</a></li>
+                <li><a href="#">Lorem ipsum dolor sit amet, consectetur adipisicing.</a></li>
+                <li><a href="#">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dolore, ut.</a></li>
+                <li><a href="#">Lorem ipsum dolor sit amet, consectetur.</a></li>
+                <li><a href="#">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</a></li>
+                <li><a href="#">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</a></li>
+                <li><a href="#">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</a></li>
+            </ul>
         </div>
+      </div>
+
     </div>
     <div class="footer">
-        <div class="wrapper">
-            <div class="columns">
-                <div class="col">
-                    &copy; 2018 7SIGNAL Solutions Inc. All Rights Reserved.
-                </div>
-                <div class="col social">
+      <div class="wrapper">
+        <div class="columns">
 
-                </div>
+          <div class="copyright">
+              &copy; 2018 7SIGNAL Solutions Inc. All Rights Reserved.
+          </div>
+
+          <div class="social">
```

Comment:
We'll want to rearrange the HTML on these slightly. Notice how your icons don't link anywhere. You need to wrap your <i> in an <a> tag and you have it the other way around. The links don't work because you have an empty link inside an icon. Also, there really isn't a need for wrapping each icon in a div. This would be my footer HTML ```    <div class="footer">
      <div class="wrapper">
        <div class="columns">
          <div class="col copyright">
              &copy; 2018 7SIGNAL Solutions Inc. All Rights Reserved.
          </div>
          <div class="col social">
            <a href="#"><i class="icon-facebook"></i></a>
            <a href="#"><i class="icon-twitter"></i></a>
            <a href="#"><i class="icon-youtube"></i></a>
            <a href="#"><i class="icon-google-plus"></i></a>
            <a href="#"><i class="icon-linkedin"></i></a>
        </div>
      </div>
    </div>
```

Once that is in place, you can style the icons with `.footer .social a { float: right; ... }`